app/src/lib/stores/helpers/pull-request-updater.ts

```diff
@@ -45,6 +45,11 @@ export class PullRequestUpdater {
 
   /** Starts the updater */
   public start() {
+    const githubRepo = forceUnwrap(
+      'Can only fetch pull requests for GitHub repositories',
```

Comment:
:lipstick: this is about updating PR information rather than just fetching, right?

Reply:
In this case, it's just about fetching so that we grab new PRs created on dotcom. I suppose this can also be seen as updating PR info (that is _why_ we do the fetch in the first place).

Reply:
I'm nit-picking the mixing of Git concepts and API concepts here, and for a message that's essentially for developers it's not worth bikeshedding. As long as this message is unique enough to be able to trace if we see it in the wild, I'm :thumbsup: